server/planning/common.py

```diff
@@ -425,19 +428,29 @@ def get_version_item_for_post(item):
 
 
 @celery.task(soft_time_limit=600)
-def enqueue_planning_item(id):
+async def enqueue_planning_item(id):
     """
     Get the version of the item to be published from the planning versions collection and enqueue it.
 
     :param id:
     :return:
     """
-    planning_version = get_resource_service("published_planning").find_one(req=None, _id=id)
+    planning_version = await get_resource_service("published_planning").find_one_async(req=None, _id=id)
+
     if planning_version:
-        try:
-            get_enqueue_service("publish").enqueue_item(planning_version.get("published_item"), "event")
-        except Exception:
-            logger.exception("Failed to queue {} item {}".format(planning_version.get("type"), id))
+        item = planning_version.get("published_item")
+        publish_response = await publish_item(
+            PublishRequest(
+                item=item,
+                item_id=item[ID_FIELD],
+                item_type="event",
```

Comment:
I know the old code always sent `event` as the type. It might be good if we send the actual item type with this publish request.